package.json

```diff
@@ -41,44 +41,44 @@
     "installPlugin": "npm install && npm run clean && npm run build && zowe plugins install .",
     "server:start": "cd server && json-server db.json --routes routes.json --ro",
     "typedoc": "typedoc --out ./docs/typedoc/ ./src/ --disableOutputCheck",
-    "prepare": "husky install"
+    "prepare": "husky"
   },
   "imperative": {
     "configurationModule": "lib/pluginDef.js"
   },
   "dependencies": {
-    "@zowe/core-for-zowe-sdk": "^7.23.2",
-    "@zowe/zosmf-for-zowe-sdk": "^7.23.2"
+    "@zowe/core-for-zowe-sdk": "^7.29.2",
```

Comment:
Should our sample remain a V2 sample, or should it be a V3 sample?

Reply:
I don't mind moving things into a zowe-v2-lts branch after this PR is merged 🙏